### PR TITLE
Added "About" button

### PR DESCRIPTION
**Pull request by davidebeatrici**
Like in Mumble, it displays the plugin's about message, if set.

### CODE REVIEW

File: mumblepahelper.cpp [hunk 1]
```diff
@@ -130,6 +130,26 @@ void MumblePAHelper::on_plugins_PluginList(const QList<PluginInfo*> plist) {
     }
 }
 
+void MumblePAHelper::on_qpbAbout_clicked(bool) {
+	QListWidgetItem *item = qlwPlugins->currentItem();
+	if(item) {
```
Comment (mkrautz) [inline mumblepahelper.cpp]: Coding style. Should be `if (item) {`...

File: mumblepahelper.cpp [hunk 2]
```diff
@@ -130,6 +130,26 @@ void MumblePAHelper::on_plugins_PluginList(const QList<PluginInfo*> plist) {
...
+	QString filename = item->data(Qt::UserRole).toString();
```
Comment (mkrautz) [inline mumblepahelper.cpp]: Indentation is wrong inside the if () {...

File: mumblepahelper.cpp [hunk 3]
```diff
@@ -130,6 +130,26 @@ void MumblePAHelper::on_plugins_PluginList(const QList<PluginInfo*> plist) {
...
+	QReadLocker lock(&plugins->qrwlPlugins);
+	foreach (PluginInfo *pi, plugins->qlPlugins) {
+		if (pi->filename == filename) {
+		lock.unlock();
+		if (pi->p->about)
```
Comment (mkrautz) [inline mumblepahelper.cpp]: Please add {} around this if.

File: mumblepahelper.cpp [hunk 4]
```diff
@@ -130,6 +130,26 @@ void MumblePAHelper::on_plugins_PluginList(const QList<PluginInfo*> plist) {
...
 }
 
+void MumblePAHelper::on_qpbAbout_clicked(bool) {
+	QListWidgetItem *item = qlwPlugins->currentItem();
+	if(item) {
+	QString filename = item->data(Qt::UserRole).toString();
+	QReadLocker lock(&plugins->qrwlPlugins);
+	foreach (PluginInfo *pi, plugins->qlPlugins) {
+		if (pi->filename == filename) {
+		lock.unlock();
```
Comment (mkrautz) [inline mumblepahelper.cpp]: Indentation is wrong inside the if () {...

**Comment by mkrautz** [comment]
Also, did the items change position? Screenshot? :)

**Comment by davidebeatrici** [comment]
mumblepahelper

**Comment by mkrautz** [comment]
Merged.

**Pull request closed without merging**